packages/inquirer-gui/src/packages/QuestionCheckbox.vue

```diff
@@ -43,4 +44,27 @@ export default {
 .v-messages {
   min-height: 0px;
 }
+div#question-checkbox-v-card {
+  border: none !important;
+}
+div#question-checkbox-v-list-item {
+  padding-left: 0 !important;
+  --v-hover-opacity: 0 !important;
+}
+form.inquirer-gui div.v-list,
+div[role="listbox"].v-theme--light.v-list {
+  background-color: var(--vscode-editor-background, var(--v-theme-background), #1f1f1f) !important;
+}
+.nobg,
+div.v-list-item--active {
+  background-color: var(--vscode-editor-background, var(--v-theme-background), #1f1f1f) !important;
+  --v-activated-opacity: 0 !important;
+  --v-hover-opacity: 0 !important;
+}
+div.v-list-item__spacer {
+  display: none !important;
+}
+div.v-list-item-action.v-list-item-action--start {
+  margin-right: 0 !important;
+}
```

Comment:
overall - massive use of `!important` annotation is questionable -> more specific selectors should be used instead. consider refactoring.